docs/website/docs/general-usage/schema.md

```diff
@@ -212,12 +212,12 @@ explicitly on a column or by a source/resource. Normalizers do not infer this hi
 naive timestamps will **always be considered as UTC**, system timezone settings are ignored by `dlt`
 :::
 
-Ultimately destination will interpret the timestamp values. Some destinations:
-- do not support naive timestamps (ie. BigQuery) and will interpret them as naive UTC by attaching UTC timezone
-- do not support tz-aware timestamps (ie. Dremio, Athena) and will strip timezones from timestamps being loaded
+Ultimately, the destination will interpret the timestamp values. Some destinations:
```

Comment:
These destination specific things should be on the destination docs page. I have had the idea of generating a section on each destination page from the most important parts of the Capabilities so it gets maintained automatically. But maybe this is overkill.